packages/salesforcedx-vscode-core/src/commands/forceApexLogGet.ts

```diff
@@ -68,6 +69,7 @@ export class ForceApexLogGetExecutor extends SfdxCommandletExecutor<
       cwd: vscode.workspace.rootPath
     }).execute(cancellationToken);
     this.attachExecution(execution, cancellationTokenSource, cancellationToken);
+    this.logMetric(execution.command.logName);
```

Comment:
Directly calling logMetric since we are not leveraging `SfdxCommandletExecutor`.